app/src/main/java/org/simple/clinic/drugs/selectionv2/entry/CustomPrescriptionEntrySheetv2.kt

```diff
@@ -84,9 +91,11 @@ class CustomPrescriptionEntrySheetv2 : BottomSheetActivity() {
 
     return RxView.clicks(saveButton)
         .mergeWith(dosageImeClicks)
-        .map { SaveCustomPrescriptionClicked() }
+        .map { SaveCustomPrescriptionClicked }
   }
 
+  private fun removeClicks(): Observable<UiEvent> = RxView.clicks(removeMedicineButton).map { RemoveCustomPrescriptionClicked }
```

Comment:
Can you break this line into multiple lines? Multiple operators on different lines please.

```kotlin
  private fun removeClicks(): Observable<UiEvent> =
      RxView
          .clicks(removeMedicineButton)
          .map { RemoveCustomPrescriptionClicked }
```

Reply:
~rule~ guideline 😅

Reply:
I just think there's too much happening here in a single line.